src/test/java/com/example/eventorganizer/event/EventServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package com.example.eventorganizer.event;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import com.example.eventorganizer.user.User;
+import com.example.eventorganizer.user.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class EventServiceTest {
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Mock
+    private EventRepository eventRepository;
+
+    @InjectMocks
+    private EventService eventService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    public void testCreateEvent() {
```

Comment:
we need cases for all custom exceptions. when user not found, when event not found and when not an admin tries to update event